sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBase.java

```diff
@@ -354,6 +354,11 @@ public SdlMsgVersion getSdlMsgVersion(){
 			return null;
 		}
 
+		@Override
+		public com.smartdevicelink.util.Version getProtocolVersion() {
+			return new com.smartdevicelink.util.Version(Byte.toString(SdlProxyBase.this.getWiProVersion()) + ".0.0");

```

Comment:
I believe we can access this information directly from wiproprotocol and should not just use the major byte as previously used. https://github.com/smartdevicelink/sdl_android/blob/master/sdl_android/src/main/java/com/smartdevicelink/protocol/WiProProtocol.java#L96